aiidalab/appstore.py

```diff
@@ -115,36 +115,17 @@ def change_vis_list(self, _=None):
 
     def render(self, _=None):
         """Show all the available apps."""
```

Comment:
I still think it is nice to keep a short description in App Store and give a longer one in "Manage app window".

Reply:
That's totally fine, I guess I just didn't get around to reimplementing that.